radiation/python/radlw/rtrnmc_gt4py.py

```diff
@@ -0,0 +1,584 @@
+import gt4py
+import os
+import sys
+import time
+import numpy as np
+import gt4py.gtscript as gtscript
+from gt4py.gtscript import FORWARD, BACKWARD, PARALLEL, Field, computation, interval, stencil, exp
+sys.path.insert(0, '/Users/AndrewP/Documents/work/physics_standalone/radiation/python')
+from config import *
+from util import create_storage_from_array, create_storage_zeros, compare_data
+from radlw_param import ngb, bpade, ntbl, eps, wtdiff, fluxfac, heatfac
+
+os.environ["DYLD_LIBRARY_PATH"]="/Users/AndrewP/Documents/code/serialbox2/install/lib"
+
+SERIALBOX_DIR = "/Users/AndrewP/Documents/code/serialbox2/install"
+sys.path.append(SERIALBOX_DIR + "/python")
+import serialbox as ser
+
+rebuild = False
+validate = False
+backend = "gtc:gt:cpu_ifirst"
+
+invars = ['semiss', 'delp', 'cldfmc', 'taucld', 'tautot', 'pklay', 'pklev',
+          'fracs', 'secdiff', 'nlay', 'nlp1', 'exp_tbl', 'tau_tbl', 'tfn_tbl',
+          'totuflux', 'totdflux', 'htr', 'totuclfl', 'totdclfl', 'htrcl', 'htrb']
+
+ddir = '/Users/AndrewP/Documents/work/physics_standalone/radiation/fortran/radlw/dump'
+serializer = ser.Serializer(ser.OpenModeKind.Read, ddir, 'Serialized_rank0')
+
+savepoints = serializer.savepoint_list()
+
+indict = dict()
+for var in invars:
+    tmp = serializer.read(var, savepoints[14])
```

Comment:
it might be difficult to keep track of savepoints by array indices, especially if you change how many savepoints there are..  it is possible to grab savepoints by their name

Reply:
Fixed.